src/NodaTime.Demo/OffsetDemo.cs

```diff
@@ -62,6 +62,58 @@ public void ConstructionFromTimeSpan()
             Assert.AreEqual(5400, offset.Seconds);
         }
 
+        [Test]
+        public void Add()
+        {
+            var lefHandOffset = Offset.FromHours(5);
+            var rightHandOffset = Offset.FromHours(6);
+
+            var result = Snippet.For(Offset.Add(lefHandOffset, rightHandOffset));
+
+            Assert.AreEqual(result, lefHandOffset + rightHandOffset);
```

Comment:
We need `result` to be on the RHS - basically this will currently be replaced with

```csharp
Console.WriteLine(leftHandOffset + rightHandOffset);
```

whereas we want to print `result`.

Reply:
In order to make the expected result more obvious, I'd probably use `Offset.FromHours(11)` too:

```csharp
Assert.AreEqual(Offset.FromHours(11), result);
```